contracts/MemeFactory.sol

```diff
@@ -59,12 +62,12 @@ interface IERC20 {
 ///           $SMTH to be burned.
 abstract contract MemeFactory {
     event OLASJourneyToAscendance(uint256 amount);
-    event Summoned(address indexed summoner, uint256 indexed memeNonce, uint256 nativeTokenContributed);
+    event Summoned(address indexed summoner, uint256 indexed memeNonce, uint256 amount);
     event Hearted(address indexed hearter, uint256 indexed memeNonce, uint256 amount);
-    event Unleashed(address indexed unleasher, address indexed memeToken, uint256 indexed lpTokenId,
-        uint256 liquidity, uint256  nativeAmountForOLASBurn);
+    event Unleashed(address indexed unleasher, uint256 indexed memeNonce, address indexed memeToken, uint256 lpTokenId,
```

Comment:
Returning nonce here allows for simpler off-chain correlation of nonce and token